src/LeftNav/LeftNav.less

```diff
@@ -0,0 +1,80 @@
+@import (reference) "../less/index.less";
+
+.LeftNav {
+  .shade(color, @primary_blue, 2);
+  background: @neutral_off_white;
+  user-select: none;
+  .text--light;
+  // Maintain computed width based on content
+  // (i.e. don't let the NavLink labels wrap)
+  .flexShrink(0);
+}
+
+.LeftNav--topnav {
+  .list-reset;
+  min-width: 9rem;
+  box-shadow: @borderWidthM 0 @borderWidthM @neutral_silver;
+  .zIndex--1; // Ensure the shadow goes over the subnav
+}
+
+.LeftNav--subnav {
+  .list-reset;
+  .tint(background, @primary_blue, 6);
+  color: @neutral_black;
+  min-width: 9rem;
+  box-shadow: @borderWidthM 0 @borderWidthM @neutral_silver;
+}
+
+.NavLink {
+  .padding--left--m;
+  .padding--right--xl;
+  cursor: pointer;
+  position: relative;
+
+  height: @size_3xl + @borderWidthS;
+  .border--bottom--s(@neutral_silver);
+}
+
+.LeftNav--subnav .NavLink {
+  .padding--left--s;
+  .padding--right--l;
+  .NavLink--icon {
+    .padding--none;
+  }
+}
+
+.NavLink--selected,
+.NavGroup--open {
+  .tint(background, @primary_blue, 6);
+  .shade(color, @primary_blue, 3);
+}
+
+.NavGroup--open {
+  .padding--top--none;
+  height: @size_3xl - @borderWidthS;
+  .border--top--s(@neutral_silver);
+  .border--bottom--m(@neutral_silver);
+  /*.text--regular;*/
+  position: relative;
+}
+
+
+.NavLink--selected,
+.NavLink:hover {
+  box-shadow: inset @borderWidthL 0 @primary_blue;
+  .zIndex--2; // For subnav links, ensure this goes over the shadow from the topnav
+}
+
+.NavLink--icon {
+  .padding--right--s;
+}
+
+.LeftNav--collapsed {
+  min-width: 0;
+  .NavLink {
+    .padding--right--s;
+  }
+  .NavLink--label {
+    display: none;
```

Comment:
this seems simpler to me for now. unless you have a strong reason to change it now, i'd rather just leave it since it works. but i'll keep that in mind if i come back and add animation for sure